### ISSUE TITLE
Discriminated union syntax: Should mention that the 1st pipe character is optional

### ISSUE THREAD

**Issue by sk84000** [dotnet/docs#21971]
In the Syntax section at the beginning, it is important to state that the first 1st pipe character is optional.
Otherwise, it is not clear that "type X = Y of Z" is valid syntax for defining a single-case discriminated union.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6020749e-d91e-8a1b-5366-25b212baa31d
* Version Independent ID: 0f28f508-dc0f-c272-383e-ec768759fe16
* Content: [Discriminated Unions - F#](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/discriminated-unions)
* Content Source: [docs/fsharp/language-reference/discriminated-unions.md](https://github.com/dotnet/docs/blob/master/docs/fsharp/language-reference/discriminated-unions.md)
* Product: **dotnet-fsharp**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**

---
[Associated WorkItem - 492217](https://dev.azure.com/msft-skilling/Content/_workitems/edit/492217)

**Comment by cartermp** [comment]
Good catch. There should probably also be a small section explaining single case, why you'd use it, and an example. Would you be willing to submit a PR? We'd be happy to review it.
